invenio_previewer/assets/js/previewer_theme.js

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
I added this file that it will generate `previewer_theme.js` and `previewer_theme.css`webpack bundles!